R/plotconfiguration-axis.R

```diff
@@ -1,3 +1,13 @@
+#' How should comparison of entities be performed
+#'
+#' @include enum.R
+#' @export
+Scaling <- enum(c(

```

Comment:
I have defined the enum Scaling. This should now be used in the code. I did not know how to map it (e.g. I have no idea what identity means). We can discuss about this